src/widgets/splits/SplitHeader.cpp

```diff
@@ -916,10 +916,6 @@ void SplitHeader::themeChangedEvent()
     }
 }
 
-void SplitHeader::moveSplit()
-{
-}
-
```

Comment:
Start a separate issue/discussion for that - out of scope image ![](https://user-images.githubusercontent.com/4962764/128595809-f7e5e42d-6352-415b-8c3e-29b1ffe8e624.png)

Reply:
No
I do not think so